lib/req-res.js

```diff
@@ -36,16 +63,32 @@ module.exports = function ReqRes ( Rabbit ) {
 			}, null );
 		}
 
+		var listener = onErrorEmit( this, log, callback ); // Listen to error emitted
+
+		log.info( {
```

Comment:
log object is always created regardless if there was logger attached or not. Log object will morph into a `fake` logger when no logger is attached